packages/app/src/explorer/Explorer.tsx

```diff
@@ -18,21 +18,25 @@ function Explorer(props: Props) {
   const ref = useRef<HTMLDivElement>(null);
   const btnRef = useRef<HTMLButtonElement>(null);
 
-  useKeyboardSupport(ref);
-
   useEffect(() => {
     btnRef.current?.focus();
   }, [btnRef]);
 
   return (
-    <div ref={ref} className={styles.explorer} role="tree">
+    <div ref={ref} className={styles.explorer} role="tree" id={EXPLORER_ID}>
```

Comment:
To get the list of buttons in the Explorer, I need to have a ref to it. As the handlers are deeper, I attach an `id` here to be able to get the reference via `document.querySelector`. We could prop-drill the `ref` or use a context instead.

Reply:
Neh, this is good. The coupling is clear thanks to the shared constant. 👍